common.py

```diff
@@ -0,0 +1,27 @@
+from github import Github
+import re
+
+# Get all issue numbers related to a PR.
+def get_related_issues(pr):
+    # Regex to pick out closing keywords.
+    regex = re.compile("(close[sd]?|fix|fixe[sd]?|resolve[sd]?)\s*:?\s+#(\d+)", re.I)
+
+    # Extract all associated issues from closing keyword in PR
+    for verb, num in regex.findall(pr.body):
+        yield int(num)
+
+    # Extract all associated issues from PR commit messages
+    for c in pr.get_commits():
+        for verb, num in regex.findall(c.commit.message):
+            yield int(num)
+
+# Takes a project as input and caches all cards in a dictionary for easy lookup
+# (and to avoid redundant network requests).
+def cache_board(project):
+    card_cache = {}
+    for column in project.get_columns():
+            for card in column.get_cards():
+                content = card.get_content()
+                if content:
+                    card_cache[content.id] = (card, column)
+    return card_cache
```

Comment:
This file will never be copied to the container, so execution of enarx-copy-labels-linked will fail.

Reply:
Wouldn't this be passed through memory, though? `enarx-copy-linked-labels` calls this function and stores the output dictionary as a variable. That shouldn't result in any change against earlier versions where this code was in the same file, right?